package.json

```diff
@@ -69,7 +69,12 @@
     "@sideway/formula": "^3.0.1",
     "@types/react": "17.0.47",
     "**/@angular-devkit/build-angular/minimatch": "3.0.5",
+    "**/@angular-devkit/build-angular/semver": "7.5.4",
+    "**/@angular/cli/semver": "7.5.4",
     "**/@angular-devkit/build-angular/webpack": "^5.76.0",
+    "**/@changesets/cli/**/semver": "7.5.4",
+    "**/@cucumber/cucumber/**/semver": "7.5.4",
+    "**/@size-limit/file/**/semver": "7.5.4",
```

Comment:
Can we bump size limit instead?

 https://github.com/ai/size-limit/commit/96d5d06cb91f83c9efb0a5ef1b4186120a9a1775#diff-e05689f8c76d55d8c7b417fd0312152c6b1ec6df305dcc3a3561321131c0a5ecR23